src/modules/test/main.c

```diff
@@ -469,17 +476,62 @@ long long CurrentMilliseconds()
     return milliseconds;
 }
 
+const char* GetStepTypeString(STEP_TYPE type)
+{
+    switch (type)
+    {
+    case COMMAND:
+        return "Command";
+    case TEST:
+        return "Test";
+    case MODULE:
+        return "Module";
+    default:
+        return "Unknown";
+    }
+}
+
+char* GetStepName(const STEP* step)
+{
+    char* name = NULL;
+
+    if (step != NULL)
+    {
+        switch (step->type)
+        {
+        case COMMAND:
+            name = strdup(step->data.command.arguments);
+            break;
+        case TEST:
+            name = calloc(strlen(step->data.test.component) + strlen(step->data.test.object) + 2, sizeof(char));
+            if (name != NULL)
+            {
+                sprintf(name, "%s.%s", step->data.test.component, step->data.test.object);
+            }
+            break;
+        case MODULE:
+            name = strdup(step->data.module.name);
+            break;
+        default:
+            break;
```

Comment:
>break; [](http://example.com/codeflow?start=12&length=6)

This break is not necessary. Neither the default label is (and the break for case MODULE)